src/daemon/pex/pex.go

```diff
@@ -218,9 +218,17 @@ func New(cfg Config, defaultConns []string) (*Pex, error) {
 
 	// Load peers from disk
 	if err := pex.load(); err != nil {
+		logger.Errorf("Error loading peers from file: %v", err)
 		return nil, err
 	}
 
+	// If no default connections then all connections are untrusted.
```

Comment:
It should read a config value, not rely upon a side effect of having the config value set in another file

Reply:
Let's say a node is started with disable-trusted-peers. Then peers are loaded from cache. Right now these peers are set as not trusted. 
Should this be specified in a command line argument? 
I mean, is it necessary an argument to say if these loaded peers are trusted or not?